### ISSUE TITLE
Success Criterion 2.6.1 Motion Actuation - SC includes user motion, but understanding document does not

### ISSUE THREAD

**Issue by SuzanneTaylor** [w3c/wcag21#620]
It sounds like the SC includes situations where the device is observing, and the user is moving without necessarily moving the device. (like Leap Motion) If so, this should be added to the Understanding document. (If this situation will wait to be addressed in a later version of the guidelines, then the SC should be edited to remove "or user motion".)

**Comment by joshueoconnor** [comment]
Relates to https://www.w3.org/TR/WCAG21/#motion-actuation and #621

**Comment by awkawk** [comment]
(Official WG Response)
Thank you for your comment. You propose to remove 'user motion' to exclude situations where the user is moving and the device just observing. The aim of the user motion part of the SC was to address motions like hand gestures that are observed by the camera or other sensors that can trigger certain events (such as forwarding a presentation, or panning a view). We have changed the SC text and the understanding text to clarify that 'user motion' means explicit actuation in the context of this SC. We have spelled out that scenarios where the user moves without explixitly actuating content (as in your your Leap Motion example) or where the device just observes without the user intentionally gesturing towards the device, are not in scope of this SC.

We will mark this issue as "defer" to ensure that it is reviewed at an appropriate time in the future.
